python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -3400,3 +3400,9 @@ def backfill_log_storage_enabled(self) -> bool:
 
     def da_request_backfills(self) -> bool:
         return False
+
+    def dagster_observe_supported(self) -> bool:
```

Comment:
What does this property do? Unclear to me right now when it's turned on vs off. Sorry if this info is elsewhere in the stack